sympy/simplify/tests/test_simplify.py

```diff
@@ -805,11 +805,10 @@ def test_issue_15965():
     A = Sum(z*x**y, (x, 1, a))
     anew = z*Sum(x**y, (x, 1, a))
     B = Integral(x*y, x)
-    bdo = x**2*y/2
+    bdo = y*Integral(x, x)
```

Comment:
Apparently y*Integral(x, x) is considered to be a simpler expression than x**2*y/2 (2 vs 3 count_ops).